src/SablierV2LockupLinear.sol

```diff
@@ -485,6 +263,11 @@ contract SablierV2LockupLinear is
             wasCanceled: false
         });
 
+        // Effects: set the cliff time if it is greater than 0.
+        if (params.range.cliff > 0) {
```

Comment:
> Removing it could save 50 gas

how did you test this? 

i also tested and the result was the other way, see also [this comment](https://github.com/sablier-labs/v2-core/discussions/805#discussioncomment-8306687)

Reply:
You are right. I only compared the gas usage of `createWithDurations` and `createWithTimestamps` for foundry run tests which is just the avg not the gas comparison when cliff is 0.